src/ui/CategoryFacet/CategoryValue.ts

```diff
@@ -83,7 +83,8 @@ export class CategoryValue implements CategoryValueParent {
     const element = $$(this.element.find('.coveo-category-facet-value-label'));
     element.addClass('coveo-selectable');
 
-    const countLabel = l('ResultCount', this.categoryValueDescriptor.count.toString());
+    const count = this.categoryValueDescriptor.count;
+    const countLabel = l('ResultCount', count.toString(), count);
```

Comment:
Though it's a small change, I prefer to include a unit test to document and enforce the Deque recommendation that we paid for.

Reply:
Good point, I'll add unit tests for it.